rocketmq-cpp/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ if(LIBEVENT_FOUND)
 endif()
 
 #set(JSONCPP_INCLUDE_DIRS C:/jsoncpp-0.10.6/include)
-set(JSONCPP_USE_STATIC_LIBS    ON) # only find static libs
+set(JSONCPP_USE_STATIC_LIBS    OFF) # only find static libs
```

Comment:
only find static libs?
I noticed you turn it off. Maybe new comments here.

Reply:
Recommend to use dynamic library.